addons/website_slides/models/slide_channel.py

```diff
@@ -601,3 +611,12 @@ def _resequence_slides(self, slide):
 
     def get_backend_menu_id(self):
         return self.env.ref('website_slides.website_slides_menu_root').id
+
+    def send_email_to_responsible(self):
```

Comment:
I would put some restrictions on this one to limit a bit calls to that method, like
  * current user is not a member of the course;
  * course is published;